adi/adf4371.py

```diff
@@ -174,3 +174,8 @@ def rf32_frequency(self):
     def rf32_frequency(self, value):
         """Get/Set the frequency of the 32GHz RF output"""
         self._set_iio_attr("altvoltage3", "frequency", True, value, self._ctrl)
+
+    @property
+    def temperature(self):
+        """Get the temperature reading"""
+        return self._get_iio_attr("temp0", "raw", False)
```

Comment:
@mhennerich is this the correct dev+attr name?

Reply:
The driver uses IIO_CHAN_INFO_PROCESSED
https://github.com/analogdevicesinc/linux/blob/master/drivers/iio/frequency/adf4371.c#L764

So the full channel name is: **in_temp0_input**

So this should be 
return self._get_iio_attr("temp0", "input", False)